src/PublicApiAnalyzers/Core/Analyzers/PublicApiAnalyzerResources.resx

```diff
@@ -126,6 +126,15 @@
   <data name="DeclarePublicApiDescription" xml:space="preserve">
     <value>All public types and members should be declared in PublicAPI.txt. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.</value>
   </data>
+  <data name="AnnotatePublicApiTitle" xml:space="preserve">
+    <value>Annotate public types and members in the declared API</value>
+  </data>
+  <data name="AnnotatePublicApiMessage" xml:space="preserve">
+    <value>Symbol '{0}' is not annotated in the declared API.</value>
```

Comment:
Nit: Does the message need to have the term "nullable" or "nullability" somewhere for clarity?